docs/source/markdown/podman-build.unit.5.md.in

```diff
@@ -0,0 +1,229 @@
+% podman-build.unit(5)
+
+# NAME
+
+podman\-build.unit - systemd unit files for building container images using Podman Quadlet
+
+# SYNOPSIS
+
+*name*.build
+
+# DESCRIPTION
+
+Build units (`.build` files) are used by **Podman Quadlet** to declaratively define systemd services that
+build container images from a `Containerfile` or `Dockerfile`.
+
+These units ensure that the image is built on the host before being used by containers or volumes. If the
+image already exists and the context hasn’t changed, subsequent executions will complete quickly thanks to
+Podman’s build cache.
+
+They are especially useful for:
+- Creating images not available in registries
+- Automating image builds during boot
+- Local testing and development pipelines
+
+A minimal `.build` file must specify:
+- `ImageTag=` — to name the built image
+- Either `File=` or `SetWorkingDirectory=`
+
+
+# USAGE SUMMARY
+
+The `.build` file is parsed by the `podman-system-generator` at boot or reload, generating a systemd
+`.service` that runs `podman build`. That service can be managed like any other unit:
+
+```bash
+systemctl --user start myimage-build.service
+```
+
+The resulting image can be referenced by `.container` or `.volume` units via:
+
+```ini
+Image=myimage.build
+```
+
+
+# FILE LOCATIONS
+
+Place `.build` files in one of the following:
+
+### Rootless
+
+- `$XDG_RUNTIME_DIR/containers/systemd/`
+- `$XDG_CONFIG_HOME/containers/systemd/` or `~/.config/containers/systemd/`
+- `/etc/containers/systemd/users/$(UID)`
+- `/etc/containers/systemd/users/`
+
+### Rootful
+
+- `/run/containers/systemd/`
+- `/etc/containers/systemd/`
+- `/usr/share/containers/systemd/`
+
```

Comment:
That is duplication you have in any file, IMO it is enough to document this only once in the main file